_docs/supportforlivesharecodewithme.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Support for Live Share / Code With Me
+styleid: sub-nav
+subtitle: Seamlessly edit files together in real-time, using IDE specific collaboration tools
```

Comment:
Leverage IDE specific collaboration tools for a fully-fledged pair programming experience